src/ansys/fluent/core/exceptions.py

```diff
@@ -0,0 +1,13 @@
+"""Custom common higher level exceptions."""
+
+
+class DisallowedValuesError(ValueError):
+    """Raises exception if argument value is not in allowed values."""
```

Comment:
Quick suggestion for the docstrings, I think this would work better (same for all other exception docstrings):
```suggestion
    """Exception raised if argument value is not in allowed values."""
```

Reply:
@raph-luc 

We are following doc guidelines.

Documentation guidelines are as follows - 

1. Method docstring begins with a verb without an "s" or "es". 

2. Class descriptions, on the other hand, begin with verbs ending with "s" or "es".